utilities/test_suite/HOST/runAudioTests.py

```diff
@@ -0,0 +1,294 @@
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+# THE SOFTWARE.
+
+import os
+import subprocess  # nosec
+import argparse
+import sys
+import datetime
+import shutil
+
+# Set the timestamp
+timestamp = datetime.datetime.now().strftime("%Y-%m-%d_%H-%M-%S")
+
+cwd = os.getcwd()
+inFilePath = os.path.join(os.path.dirname(cwd), 'TEST_AUDIO_FILES', 'eight_samples_single_channel_src1')
+
+# Checks if the folder path is empty, or is it a root folder, or if it exists, and remove its contents
+def validate_and_remove_files(path):
+    if not path:  # check if a string is empty
+        print("Folder path is empty.")
+        exit()
+
+    elif path == "/*":  # check if the root directory is passed to the function
+        print("Root folder cannot be deleted.")
+        exit()
+
+    elif os.path.exists(path):  # check if the folder exists
+        # Get a list of files and directories within the specified path
+        items = os.listdir(path)
+
+        if items:
+            # The directory is not empty, delete its contents
+            for item in items:
+                item_path = os.path.join(path, item)
+                if os.path.isfile(item_path):
+                    os.remove(item_path)
+                elif os.path.isdir(item_path):
+                    shutil.rmtree(item_path)     # Delete the directory if it exists
+
+    else:
+        print("Path is invalid or does not exist.")
+        exit()
+
+# Check if the folder is the root folder or exists, and remove the specified subfolders
+def validate_and_remove_folders(path, folder):
+    if path == "/*":  # check if the root directory is passed to the function
+        print("Root folder cannot be deleted.")
+        exit()
+    if path and os.path.isdir(path + "/.."):  # checks if directory string is not empty and it exists
+        output_folders = [folder_name for folder_name in os.listdir(path + "/..") if folder_name.startswith(folder)]
+
+        # Loop through each directory and delete it only if it exists
+        for folder_name in output_folders:
+            folder_path = os.path.join(path, "..", folder_name)
+            if os.path.isdir(folder_path):
+                shutil.rmtree(folder_path)  # Delete the directory if it exists
+                print("Deleted directory:", folder_path)
+            else:
+                print("Directory not found:", folder_path)
+
+# Validate if a path exists and is a directory
+def validate_path(input_path):
+    if not os.path.exists(input_path):
+        raise ValueError("path " + input_path +" does not exist.")
+    if not os.path.isdir(input_path):
+        raise ValueError("path " + input_path + " is not a directory.")
+
+# Get a list of log files based on a flag for preserving output
+def get_log_file_list(preserveOutput):
+    return [
+        "../../OUTPUT_AUDIO_PERFORMANCE_LOGS_HOST_" + timestamp + "/Tensor_host_audio_raw_performance_log.txt",
```

Comment:
Lets rename the file as OUTPUT_PERFORMANCE_AUDIO_XX, so that we need to add new formats in .gitignore for audio alone

Reply:
Done